internal/commands/macho/objc.go

```diff
@@ -890,8 +890,12 @@ func (o *ObjC) processForwardDeclarations(m *macho.File) (map[string]Imports, er
 	}
```

Comment:
should this TODO be addressed as well?

Reply:
Definitely! Do you mind if I do that in another PR? I was planning to do that already.

Reply:
Sounds great!